tests/system/action/topic/test_json_upload.py

```diff
@@ -195,3 +158,77 @@ def test_json_uplad_permission(self) -> None:
             {"data": [{"title": "test"}], "meeting_id": 1},
             Permissions.AgendaItem.CAN_MANAGE,
         )
+
+
+class TopicJsonUploadForUseInImport(BaseActionTestCase):
```

Comment:
Same comment here as in the account import: Move these to a separate file

Reply:
see comment in account import